apps/studio/schemas/person.ts

```diff
@@ -19,9 +20,74 @@ export default defineType({
       type: "string",
     }),
     defineField({
-      name: "linkedinURL",
-      title: "LinkedIn URL",
-      type: "url",
+      name: "email",
+      title: "Email",
+      type: "email",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "socials",
+      title: "Socials",
+      type: "array",
+      of: [
+        {
+          type: "object",
+          fields: [
+            defineField({
+              name: "platform",
+              title: "Platform",
+              type: "string",
+              options: {
+                list: ["LinkedIn", "Personal Website"],
+              },
+              validation: (Rule) => Rule.required(),
+            }),
+            defineField({
+              name: "label",
+              title: "Label",
+              description:
+                "Primary identifier of platform (i.e. Username). Include a prefix character (@, ~, etc.) if used by brand.",
+              type: "string",
+            }),
+            defineField({
+              name: "link",
+              title: "Link",
+              type: "url",
+              validation: (Rule) =>
+                Rule.required().uri({ scheme: ["http", "https", "mailto"] }),
+            }),
+          ],
+          preview: {
+            select: {
+              title: "platform",
+              platform: "platform",
+              subtitle: "label",
+              link: "link",
```

Comment:
What does link do here?

Reply:
I was trying to use the link as the subtitle of the label field was empty, so I needed link so that I could use its value in the `prepare` function

Reply:
Ah I see. In that case you probably shouldn't return it in your prepare function on line 84.

Reply:
You can also use a nullish coalescing operator to simplify it a bit if you want.
i.e. instead of `subtitle ? subtitle : link`, `subtitle ?? link`

I would change subtitle to label on line 64 to reduce confusion.

Reply:
Fixed in [`09d1d86`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/09d1d8696fd479b7a0ae77d1e6d1cd0219bddd72)